spec/features/idv/max_attempts_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+feature 'IdV max attempts' do
+  include IdvHelper
+
+  scenario 'profile shows failure modal after max attempts', :email, :idv_job, :js do
```

Comment:
Previously, we only had an example like this for the financial step. I went ahead and added an example for each step. I can tear it down if we think that is overkill